libs/execution_context/ebpf_core.c

```diff
@@ -37,7 +38,8 @@ const NPI_MODULEID ebpf_general_helper_function_module_id = {
 static ebpf_pinning_table_t* _ebpf_core_map_pinning_table = NULL;
 
 // Assume enabled until we can query it.
-static bool _ebpf_platform_hypervisor_code_integrity_enabled = true;
+// Extern variable defined in ebpf_core_jit.h
+bool _ebpf_platform_hypervisor_code_integrity_enabled = true;
```

Comment:
Coding convention is that only `static` things start with `_`
(see point 6 in https://github.com/microsoft/ebpf-for-windows/blob/main/docs/DevelopmentGuide.md#naming-conventions)